### DISCUSSION TITLE
instantiate - 기초질문이지만 답변해주시면 감사합니다

### DISCUSSION THREAD

**Discussion by janggf** [naverz/zepeto-studio-kor#516] [5. Q&A]
instantiate 안되는거같네요
오브젝트 생성 방법이 뭔가요?
덤으로 시간이 있으시다면 생성된 오브젝트를  모든 클라이언트에 보이도록 하는 간단한 예제도 부탁드립니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 질문 주신 내용에 대한 가이드는 아래 페이지에 있습니다.
https://studio.zepeto.me/kr/guides/gameobjects-and-components

간단한 예제를 보여드리면 다음과 같습니다. 
우선 Create Empty 로 스크립트를 붙여줄 오브젝트를 생성합니다. 
![스크린샷 2022-03-22 오후 4 05 26](https://user-images.githubusercontent.com/100118650/159426177-f1cca346-0ac6-4194-b2d8-c75352a7b2e4.png)

다음과 같이 시작하면 (0,0,0) 위치에 지정한 오브젝트를 생성시키는 코드를 추가합니다.
인스펙트에서 item자리에 생성시키고 싶은 프리팹 또는 오브젝트를 연결시킨 후 실행합니다.

import { GameObject, Object, Vector3 } from 'UnityEngine';
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class CreateObject extends ZepetoScriptBehaviour {

    public item: GameObject;
    
    Start() {
        var obj = Object.Instantiate(this.item);
        this.item.transform.position = new Vector3(0,0,0);
    }

}


이처럼 여러가지 기초적인 예제들을 설명하는 강의 영상 및 샘플 게임을 준비 중에 있으니 조금만 기다려 주세요~

**Reply by janggf** [reply]
너무 기초라 죄송합니다 ㅋㅋ  Object 이게 원인이였네요 이런거 모르고 진행안되면 열받아서 ㅋㅋ

**Reply by Zepeto-TechEvangelist** [reply]
import가 제일 중요합니다!
